tests/crosstest/dubbo2kitex/base_type_test.go

```diff
@@ -105,6 +105,12 @@ func TestEchoInt64(t *testing.T) {
 	assertEcho(t, err, req, resp)
 }
 
+func TestEchoFloat(t *testing.T) {
+	var req float64 = 12.3456
```

Comment:
Avoid using such values which can't not be stored precisely in float/double. `1.0` will be a better choice.

Same below.